packages/message/src/StepFunctions.ts

```diff
@@ -58,30 +63,39 @@ const pullStepFunctionEvent = async (event) => {
  * Parse step message with CMA keys and replace specified key in event with contents
  * of S3 remote message
  *
- * @param {Object} stepMessage - Message for the step
+ * @param {CMAMessage} stepMessage - Message for the step
  * @param {string} stepName - Name of the step
  * @returns {Promise<Object>} Parsed and updated event with target path replaced by remote message
  *
  * @async
  * @alias module:StepFunctions
  */
-const parseStepMessage = async (stepMessage, stepName) => {
-  let parsedStepMessage = stepMessage;
+
+type CMAMessage = {
```

Comment:
This should be exported, and should probably live somewhere else.

Reply:
ee0fc38ba7f6b29a7a39153e0e60e41406d6a5c4